src/Workspaces/Core/Portable/Shared/TestHooks/INotifyBlockedOnCompletion.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.Shared.TestHooks
+{
+    internal interface INotifyBlockedOnCompletion
```

Comment:
Please document. My first take on the name was "completion" was this was also related to the completion feature. In any other project the name might be fine but it's a bit more confusing here. :smile:

Reply:
Also before you document this, see my earlier comment where I'm describing what I think we actually *want* this interface to be.

Reply:
Yep, you was right